<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFunction</class>
 <widget class="QMainWindow" name="UserFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>render</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="outputDialog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>269</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>471</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>您可以使用以下两种方式来进行渲染：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/res/render.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/res/render.png</normaloff>:/res/render.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/res/render.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/res/uploadAndRender.png</normaloff>:/res/uploadAndRender.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>96</width>
      <height>96</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>上传文件并渲染(&amp;U)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>pushButton</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>打开桌面环境并渲染(&amp;D)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>pushButton_2</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_R">
    <property name="title">
     <string>渲染(&amp;R)</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_R"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>上传文件(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>新建渲染</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>桌面渲染(&amp;D)</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>下载文件(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>重置渲染(&amp;F)</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>注销(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
